website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md

```diff
@@ -111,3 +113,15 @@ In addition to these fields, there are two other optional fields that can be con
 
 
 <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/bigquery-connection.png" title="A valid BigQuery connection"/>
+
+### BigQuery OAuth
+**Available in:** Development environments, Enterprise plans only
+
+The OAuth auth method permits dbt Cloud to run development queries on behalf of
+a BigQuery user without the configuration of BigQuery password in dbt Cloud. For
```

Comment:
I like the idea here, but i think this line might be confusing to bigquery users since there is no notion of a "password" in bigquery! What do you think about saying "service account" here instead? It's not 100% true (you still do need to upload a keyfile) but it's _mostly_ and maybe less confusing? Take it or leave it!

```suggestion
a BigQuery user without the configuration of BigQuery service account keyfile in dbt Cloud. For
```